.ci/wait-for-services.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+readonly WAIT_FOR_IT_URL=https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh
```

Comment:
maybe it is better to make hard copy to repo?

Reply:
:+1: